contracts/debt-contracts/RepaymentManager.sol

```diff
@@ -180,7 +184,7 @@ contract RepaymentManager is Initializable, IRepaymentManager, ControllerRole {
     function _increaseShares(address account, uint256 shares_) private {
         require(account != address(0), "Account must not be zero address");
         require(shares_ > 0, "Can not increase by zero shares");
-        require(_shares[account] >= 0, "Account has zero shares");
+        // require(_shares[account] >= 0, "Account has zero shares"); // DAN: this makes no sense
```

Comment:
This line depends on the use of the `addPayee` function for new lenders, who do not have any shares as of yet.
https://github.com/enabledao/enable-contracts/blob/track-repayment-status/contracts/debt-contracts/RepaymentManager.sol#L164